webservices/src/main/java/com/javaschool/webservices/service/impl/PackageServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.javaschool.webservices.service.impl;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+
+import com.javaschool.webservices.model.PackageType;
+import com.javaschool.webservices.service.PackageService;
+import com.javaschool.webservices.service.ShippingService;
+
+@Service
+public class PackageServiceImpl implements PackageService {
+	
+	private final ShippingService shippingService;
+	
+	public PackageServiceImpl(ShippingService shippingProducerService) {
+		this.shippingService = shippingProducerService;
+	}
+
+	@Override
+	public List<String> getSizes() {
+		List<String> sizesList = new LinkedList<>();
+		
+		sizesList.add("SMALL");
```

Comment:
This code is hardcoded, I know you are going to work on it in the following stories, but you can add a TODO comment here, so that your teammates know you are going to implement this correctly in the future and to what story it is related.

Reply:
Ok, next time I won't forget to add a TODO comment.